### PR TITLE
Translate various settings information

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Translate settings information

Before
----------------------------------------
Settings info untranslated (e.g in api explorer)

After
----------------------------------------
Settings info translated. Some wording tweaks.

Technical Details
----------------------------------------

Comments
----------------------------------------

### CODE REVIEW

File: settings/Core.setting.php [hunk 1]
```diff
@@ -397,10 +397,10 @@
     ),
     'default' => 3,
     'add' => '4.3',
-    'title' => 'Maximum File Size (in MB)',
+    'title' => ts('Maximum File Size (in MB)'),
     'is_domain' => 1,
     'is_contact' => 0,
-    'description' => 'Maximum Size of file (documents, images, etc.) which can attached to emails or activities.<br />Note: php.ini should support this file size.',
+    'description' => ts('Maximum Size of file (documents, images, etc.) which can attached to emails or activities.<br />Note: php.ini should support this file size.'),
```
Comment (mattwire) [inline settings/Core.setting.php]: can **be** attached

File: settings/Core.setting.php [hunk 2]
```diff
@@ -378,10 +378,10 @@
     ),
     'default' => 3,
     'add' => '4.3',
-    'title' => 'Maximum Attachments',
+    'title' => ts('Maximum Attachments'),
     'is_domain' => 1,
     'is_contact' => 0,
-    'description' => 'Maximum number of files (documents, images, etc.) which can attached to emails or activities.',
+    'description' => ts('Maximum number of files (documents, images, etc.) which can attached to emails or activities.'),
```
Comment (mattwire) [inline settings/Core.setting.php]: can **be** attached

**Comment by mattwire** [comment]
Merge on pass

**Comment by mattwire** [comment]
test this please

**Comment by seamuslee001** [comment]
Jenkins re test this please

**Pull request merged**